synapse/federation/federation_server.py

```diff
@@ -543,8 +543,19 @@ def event_from_pdu_json(self, pdu_json, outlier=False):
         return event
 
     @defer.inlineCallbacks
-    def exchange_third_party_invite(self, invite):
-        ret = yield self.handler.exchange_third_party_invite(invite)
+    def exchange_third_party_invite(
```

Comment:
Being incredibly, incredibly consistent with Erik's style of layering. Even if it makes me sad.